Block/Js.php

```diff
@@ -473,6 +482,32 @@ public function isLoadConnectJsDynamic()
         
         return false;
     }
+    
+    /**
+     * If feature switch M2_DISABLE_TRACK_ON_HOME_PAGE is enabled,
+     * then the Bolt track.js should be disabled on the home page.
+     */
+    public function isDisableTrackJsOnHomePage()
+    {
+        if ($this->featureSwitches->isDisableTrackJsFromHomePage()) {
+            return true;
+        }
+        
+        return false;
+    }
+    
+    /**
+     * If feature switch M2_DISABLE_TRACK_ON_NON_BOLT_PAGES is enabled,
+     * then the Bolt track.js would be loaded only on pages where we have Bolt connect.js.
+     */
+    public function isDisableTrackJsFromNonBoltPage()
```

Comment:
```suggestion
    public function isDisableTrackJsOnNonBoltPages()
```
is semantically more correct, please change everywhere for consistency, Decider as well

Reply:
Fixed